File: bin/oncommand.py
import os
import sys
import traceback


DEBUG = False


def oncommand(cmdTag, fileName, isMultiFile):
    print('cmd name: ' + cmdTag + '\nfile: ' + fileName + '\nisMultiFiles: ' + str(isMultiFile))
    return True


def main(params):
    if len(params) < 3:
        print('invalid params')
        return False
    return oncommand(params[1], params[2], True if len(params) >= 4 else False)


if __name__ == '__main__':
    ret = False
    try:
        if DEBUG:
            ret = main([__file__, '', ''])  # 这里在调试状态下可以随意填参数，方便排查问题
        else:
            ret = main(sys.argv)
        if ret is False:
            print("failed")
    except:
        print(traceback.format_exc())
        os.system('pause')

File: bin/oncommand.lua


function main(cmdname, file, isMultiFiles)
	cmdname = cmdname or 'nil'
	file = file or 'nil'
	isMultiFiles = isMultiFiles or 'false'
	print('cmd name: '..cmdname..'\nfile: '..file..'\nisMultiFiles: '..isMultiFiles)
	os.execute('pause')
	return 1
end

return main(...)